packages/date-time-picker/test/date-time-picker.test.ts

```diff
@@ -9,25 +9,25 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
 
-import '../sp-time-field.js';
-import { TimeField } from '..';
+import { elementUpdated, expect, fixture } from '@open-wc/testing';
 import { testForLitDevWarnings } from '../../../test/testing-helpers.js';
+import { DateTimePicker } from '../src/DateTimePicker.js';
+import { html } from 'lit';
 
-describe('TimeField', () => {
+describe('DateTimePicker', () => {
     testForLitDevWarnings(
         async () =>
-            await fixture<TimeField>(
+            await fixture<DateTimePicker>(
                 html`
-                    <sp-time-field></sp-time-field>
+                    <sp-date-time-picker></sp-date-time-picker>
                 `
             )
     );
-    it('loads default time-field accessibly', async () => {
-        const el = await fixture<TimeField>(
+    it('loads default sp-date-time-picker accessibly', async () => {
```

Comment:
If you were able to stub out some tests you might expect in the future, the would be much appreciated.

```js
it('does something that Mirek built', aync () => {});
it('does something else that Mirek built', aync () => {});
```

This way we don't have to think _what_ to test in future passes, just _how_ to test them.

Reply:
I'm OK with skipping test for the near term, if you are. It seems like it would be a good idea to list out some of the things that the element does that _could_ be tested now, while the changes are less cold in your mind. This was a comment asking that we list some things that it does:
```
it('makes me a sandwich', async function() {});
// etc...
```
So that we know what needs tests when it comes time for them.